### PR TITLE
doc: Updated Readme.md

### PR DESCRIPTION
**Pull request by Logeshkumar007**
I've Added Pre-requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,10 @@
 
 These are software testing projects done during the course of my Software Testing class at SKCET. The projects are done using Selenium.
 
+### Prerequirements
```
Comment (priyanshu-t-singh) [inline README.md]: Add an emoji to it

Comment (priyanshu-t-singh) [reply README.md]: Trying using which is related to pre-requirments

**Review by Blank-09 — COMMENTED** [review]
Review done.

File: README.md (hunk 1 above)
Comment (Logeshkumar007) [reply README.md]: okay I will be working on it

**Comment by Logeshkumar007** [comment]
I have updated an emoji. Please review it.

**Review by Blank-09 — COMMENTED** [review]
Ok. You're good to go @Logeshkumar007 🎉

**Comment by Logeshkumar007** [comment]
Thank you @Blank-09. It was my pleasure to contribute with you.

**Pull request merged**